client/src/stories/AuthenticationForm.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import AuthenticationForm from '../components/AuthenticationForm';
+import styled from '@emotion/styled';
+
+export default {
+  title: 'AuthenticationForm',
+  component: AuthenticationForm,
+};
+
+const Container = styled.div`
+  background-color: darkblue;
+  padding: 20px;
+`;
+
+export const Login = () => (
+  <Container>
+    <AuthenticationForm type="login" />
```

Comment:
to be consistent with naming you should either have here 'logIn' or change the other type to 'signup'